docs/snippets/type_syntax/color.md

```diff
@@ -0,0 +1,9 @@
+The legal values for a [`<color>`](/css_types/color) depend on the [class `Color`][textual.color.Color] and include:
+
+ - a recognised [named color](../../api/color#textual.color--named-colors) (e.g., `red`);
+ - a hexadecimal number representing the RGB values of the color (e.g., `#F35573`);
+ - a color description in the RGB system (e.g., `rgb(23,78,200)`);
+ - a color description in the HSL system (e.g., `hsl(290,70%,80%)`); and
+
+For more details about the exact formats accepted, see [the class method `Color.parse`][textual.color.Color.parse].
```

Comment:
I think this page should be exhaustive regarding the color formats. Since we're talking about CSS, I don't think we should direct people to the code reference.

Reply:
Fixed in 1ef20eb.